bn_s_mp_reverse.c

```diff
@@ -6,7 +6,7 @@
 /* reverse an array, used for radix code */
 void s_mp_reverse(unsigned char *s, int len)
```

Comment:
oh I just realized that it isn't done here... can you please change `len` to be `size_t`